content/post/74-lisbon-hack-week.md

```diff
@@ -0,0 +1,30 @@
+---
+date: 2019-03-09
+url: 74-lisbon-hack-week
+tags: hack week ipfs libp2p community researchers
+title: 2018 Hack Week - Lisbon
+author: André Cruz
+---
+
+In the week of May 21, 2018, we hosted another Hack Week, yet again in the sunny city of Lisbon.
+
+The goal of this event was for community members and researchers to meet in person, spend quality time together and work towards solving concrete problems. It was incredible to see groups of people coming up with ideas and solutions for problems that had been lingering for some time. The hack spirit quickly spread and fantastic work came out of it.
+
+The majority of discussions and collaborations were around IPFS, libp2p and applications built on top of them, but we also hosted a [CRDT Research meetup](https://www.youtube.com/playlist?list=PLuhRWgmPaHtTVkko1ZTn-qcGb-n6EqHff) on the second day.
+
+We recorded many of the presentations from the event, and compiled them into a [YouTube playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtR1hDEAKYXtUguNBkg2qceQ). You can see them all here:
```

Comment:
```suggestion
Didn't have a chance to be present? No worries, we got you back. Below you can find the list of report back presentations at the end of the week so that you learn about what got developed, designed and decided. [Full playlist playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtR1hDEAKYXtUguNBkg2qceQ), List: